app/Models/CategoryDAO.php

```diff
@@ -80,9 +80,13 @@ protected function autoUpdateDb($errorInfo) {
 	}
 
 	public function addCategory($valuesTmp) {
-		$sql = 'INSERT INTO `_category`(name, attributes) '
-		     . 'SELECT * FROM (SELECT TRIM(?) AS name, TRIM(?) AS attributes) c2 '	//TRIM() to provide a type hint as text
-		     . 'WHERE NOT EXISTS (SELECT 1 FROM `_tag` WHERE name = TRIM(?))';	//No tag of the same name
+		// TRIM() to provide a type hint as text
+		// No tag of the same name
+		$sql = <<<'SQL'
+INSERT INTO `_category`(name, attributes)
+SELECT * FROM (SELECT TRIM(?) AS name, TRIM(?) AS attributes) c2
+WHERE NOT EXISTS (SELECT 1 FROM `_tag` WHERE name = TRIM(?))
+SQL;
```

Comment:
This is so much clearer :+1:

Reply:
Yes - but I actually regret having done that as part of this PR. Small detail...